GOVERNANCE.md

```diff
@@ -80,9 +80,9 @@ When a consensus cannot be found a team member can call for a majority vote on a
 Many of the day-to-day project maintenance tasks can be done by a lazy consensus model. But the following items must be called to vote:
 
 * Adding a team member (simple majority)
-* Removing a team member (super majority)
-* Changing the governance rules (this document) (super majority) 
-* Licensing and intellectual property changes (including new logos, wordmarks) (simple majority)
+* Removing a team member (supermajority)
+* Changing the governance rules (this document) (supermajority)
+* Licensing and intellectual property changes (including new logos, wordmarks) (supermajority)
```

Comment:
@ElenaFdR is this change from simple to supermajority intentional?

Reply:
From my part it is. Changing licensing and IPR may have a big impact on OpenZaak. Therefore, a simple majority is not enough.